android_xwalk_hack.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Get the directory where this script is and set ROOT_DIR to that path. This
+# allows script to be run from different directories but always act on the
+# directory of the project (which is where this script is located).
+ROOT_DIR="$(cd "$(dirname $0)"; pwd)";
+
+if [ "$1" = "debug" ]; then
+  APK_FILE="$ROOT_DIR/build/dev/uproxy/android/platforms/android/build/outputs/apk/android-armv7-debug.apk"
+elif [ "$1" = "release" ]; then
+  APK_FILE="$ROOT_DIR/build/dist/android/platforms/android/build/outputs/apk/android-armv7-release.apk"
+else
+  echo "Unknown build type $1"
+  exit 1
+fi
+
+LIB_URL="https://github.com/uProxy/uproxy/releases/download/v0.9.1/libxwalkcore.so"
```

Comment:
The changes are here:
https://github.com/bemasc/libjingle-hack
https://github.com/bemasc/webrtc-hack/commits/bemasc-uproxy-transform-xwalk19
https://github.com/bemasc/crosswalk-hack